BE/toDoListAPI/src/main/java/shcool/mindera/toDoListAPI/repository/GroupsRepositoryInterface.java

```diff
@@ -0,0 +1,15 @@
+package shcool.mindera.toDoListAPI.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import shcool.mindera.toDoListAPI.entities.GroupsEntity;
+
+import java.util.List;
+
+@Repository
+public interface GroupsRepositoryInterface extends JpaRepository<GroupsEntity, Integer> {
```

Comment:
GroupsRepositoryInterface parece redundante
```suggestion
public interface GroupsRepository extends JpaRepository<GroupsEntity, Integer> {
```

Reply:
Volto a dizer ticket de entidades, no ticket dos repositorios fazemos essas alterações

Reply:
> Volto a dizer ticket de entidades, no ticket dos repositorios fazemos essas alterações

Ticket das entidades certo, mas que criaste os repos ficam já criados direito ne